src/main/java/bio/terra/cda/app/controller/MetaApiController.java

```diff
@@ -30,16 +36,27 @@ public MetaApiController(ApplicationConfiguration applicationConfiguration) {
 
   @Override
   public ResponseEntity<SystemStatus> serviceStatus() {
-    HttpStatus httpStatus = HttpStatus.OK;
-
-    SystemStatusSystems otherSystemStatus =
-        new SystemStatusSystems().ok(true).addMessagesItem("everything is fine");
+    SystemStatus systemStatus = new SystemStatus();
+    try {
+      BigQuery bigQuery = BigQueryOptions.newBuilder().setProjectId("gdc-bq-sample").build().getService();
+      String StatusCheck = bigQuery.getDataset("cda_mvp").getDatasetId().getDataset();
+      if (StatusCheck.equals("cda_mvp")) {
```

Comment:
The logic here isn't correct. If the BQ call doesn't throw an exception but this check fails, the status will be empty.